board/aspeed/evb_ast2700/evb_ast2700.c

```diff
@@ -233,6 +235,23 @@ void aspeed_errata_app_note_6_pcie_ep()
        printf("applying Aspeed app note 6 pcie ep ... Done\n");
 }
 
+/* Sys Scratch reg that holds sys_rst info */
+#define ASPEED_SYS_SCRATCH_7FC 0x12C027FC
+
+void update_por_env(void)
+{
+       const char *s;
+       u32 por_rst = readl(ASPEED_SYS_SCRATCH_7FC);
+
+       /* set reset reason env */
+       printf("Scratch register value: 0x%08x\n", por_rst);
+       if (por_rst & (u32)0x1)
```

Comment:
you may want to define a macro for the scratch register bit mask and use it here.

Reply:
Done